demo/src/main/res/layout/merge_player_controls.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content"
+  tools:parentTag="LinearLayout">
+
+  <ImageView
+    android:id="@+id/player_controls_play_pause"
+    android:layout_width="48dp"
+    android:layout_height="48dp"
+    android:layout_marginLeft="12dp"
+    android:layout_marginRight="12dp"
+    android:src="@drawable/play" />
+
+  <TextView
+    android:id="@+id/player_controls_elapsed_time"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_gravity="center_vertical"
+    android:textColor="@android:color/white"
+    android:text="00:00"
+    tools:text="00:00" />
+
+  <SeekBar
+    android:id="@+id/player_controls_progress"
+    style="@style/Controls.ScrubBar"
+    android:layout_width="0dp"
+    android:layout_height="wrap_content"
+    android:layout_weight="100"
+    android:layout_gravity="center_vertical"
+    android:layout_marginLeft="12dp"
+    android:layout_marginRight="12dp" />
+
+  <TextView
+    android:id="@+id/player_controls_time_remaining"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_gravity="center_vertical"
+    android:layout_marginRight="12dp"
+    android:textColor="@android:color/white"
+    android:text="01:00"
+    tools:text="01:00" />
```

Comment:
Don't need this since the `android:text` has the same value

Reply:
06ff11bf3dc8a1796dbec343841174e645af210d